config/boards/khadas-vim1s.wip

```diff
@@ -2,7 +2,7 @@
 BOARD_NAME="Khadas VIM1S" # don't confuse with VIM1 (S905X)
 BOARDFAMILY="meson-s4t7"
 KERNEL_TARGET="legacy"
-BOARD_MAINTAINER="rpardini viraniac"
+BOARD_MAINTAINER="viraniac"
```

Comment:
It seem like that rpardini forgot to update the informations?

Reply:
This is very puzzling to me. Where is this change coming from? I added myself in the BOARD_MAINTAINER in the original PR...

Reply:
https://www.armbian.com/update-data/

And again, I suggest to use the datas from the repo to update our website instand to invert it.

Reply:
I have update info in database but I also shuffled some keys around yesterday ... Its bug. Ignore, will fix this when possible.

Reply:
I can help to do the generation. If the website read the json, we can combine the generated json and the origin json in the script

Reply:
As I suspected, keys are problem - have to update them on the other side too.
https://github.com/armbian/scripts/actions/runs/5830324963/job/15811478342
`get: armbian_maintainers.json: Fatal error: Host key verification failed.`